base/sysimg.jl

```diff
@@ -115,9 +115,9 @@ include("intfuncs.jl")
 include("nullable.jl")
 
 # I/O
+include("show.jl")
 include("task.jl")
 include("lock.jl")
-include("show.jl")
```

Comment:
Not doing this gave 
```
LoadError(at "sysimg.jl" line 118: LoadError(at "task.jl" line 379: Base.MethodError(f=show, args=(Base.AbstractIOBuffer{Array{UInt8, 1}}(data=Array{UInt8, 1}[], readable=true, writable=true, seekable=true, append=false, size=0, maxsize=9223372036854775807, ptr=1, mark=-1), false))))
```
If you still think it's fratuitous, I'd be happy to revert the change.

And thank you for the feedback.